tox.ini

```diff
@@ -0,0 +1,21 @@
+[tox]
+envlist = py35, py36, flake8
+skipsdist=True
+
+[travis]
+python =
+	3.5: py35, flake8
+	3.5.2: py35, flake8
```

Comment:
I have no idea. It is there in the .travis.yml so that why I kept it. The reason it is in the tox.ini is for the flake8 check.